src/cli.rs

```diff
@@ -530,6 +530,10 @@ pub struct Opt {
     #[structopt(long = "wrap-right-prefix-symbol", default_value = "…")]
     pub wrap_right_prefix_symbol: String,
 
+    #[structopt(long = "navigate-regexp", default_value = "")]
+    /// A regexp to use in the less pager when navigating (auto-generated when empty)
+    pub navigate_regexp: String,
```

Comment:
After anguished debate with myself some months ago I decided to use `regex` as opposed to `regexp` for the regular expression options in the CLI.

Reply:
Sounds good!